Sources/protoc-gen-swift/MessageGenerator.swift

```diff
@@ -314,6 +295,44 @@ class MessageGenerator {
     }
   }
 
+  func generateRuntimeSupport(printer p: inout CodePrinter, file: FileGenerator, parent: MessageGenerator?) {
+    p.print("\n")
+    p.print("// Support for the runtime.\n")
```

Comment:
Removed.  I was also debating just doing one comment before all these later decls (from the FileGenerator), but I'm not sure it's worth it, so just removing it.